pvc_to_pvs_image_import/_topics/preparing_powervc_node.md

```diff
@@ -0,0 +1,223 @@
+### Assumptions:
```

Comment:
For all the commands/sections, we should make it clear where the user is supposed to be running the commands--e.g., on their client workstation, MiQ server, PowerVC server, etc.